src/components/application_manager/include/application_manager/commands/hmi/sdl_activate_app_request.h

```diff
@@ -58,32 +58,45 @@ class SDLActivateAppRequest : public RequestFromHMI {
   /**
    * @brief SDLActivateAppRequest class destructor
    **/
-  virtual ~SDLActivateAppRequest();
+  ~SDLActivateAppRequest() OVERRIDE;
 
   /**
    * @brief Execute command
    **/
-  virtual void Run();
+  void Run() OVERRIDE;
 
   /**
    * @brief onTimeOut allows to process case when timeout has appeared
    * during request execution.
    */
-  virtual void onTimeOut();
+  void onTimeOut() OVERRIDE;
 
   /**
    * @brief on_event allows to handle events
    *
    * @param event event type that current request subscribed on.
    */
-  virtual void on_event(const event_engine::Event& event);
+  void on_event(const event_engine::Event& event) OVERRIDE;
 
  private:
   uint32_t app_id() const;
   uint32_t hmi_app_id(const smart_objects::SmartObject& so) const;
 
-  DevicesApps FindAllAppOnParticularDevice(
-      const connection_handler::DeviceHandle handle);
+  /**
+   * @brief Retrieves all v4 protocol applications for particular device
+   * @param handle Device handle
+   * @return List of applications registered over v4 protocol
+   */
+  SDL4Apps get_sdl4_apps(const connection_handler::DeviceHandle handle) const;
```

Comment:
@dev-gh For the people who is not aware of SDL  it will be ambiguous. May be add undersore or something like this: `get_sdl_v4_apps`. Because `4` can be treated as `for` or just as number of apps but not SDL version.